src/main/java/com/courtcanva/ccfranchise/services/FranchiseeService.java

```diff
@@ -115,4 +118,11 @@ public Optional<Franchisee> findFranchiseeById(Long franchiseeId) {
     }
 
 
+    public List<OpenOrderGetDto> getFirst10OpenOrders(Long id) {
+        Optional<Franchisee> franchiseeFromDatabase = franchiseeRepository.findById(id);
+        Franchisee franchisee = franchiseeFromDatabase.orElseThrow(() -> new ResourceNotFoundException(
```

Comment:
This can be shorten as
```java
Franchisee franchisee = franchiseeRepository.findById(id)
    .orElseThrow(() -> new ResourceNotFoundException("Can't find franchisee id " + id));
```

Reply:
brilliant